src/components/shared/ProfileModal.tsx

```diff
@@ -109,6 +111,9 @@ const styles: Styles = {
 
 const Shared = forwardRef<Ref, Props>((props, ref) => {
   let modal: Modal | null;
+  const [deleteFriendMutation, {
+    error: deleteFriendError, loading: deleteFriendLoading,
+  }] = useMutation<{ id: string }, MutationDeleteFriend>(MUTATION_DELETE_FRIEND);
```

Comment:
I found the way how to refetch and get friend data without using `useFriendContext`.
It seems to work well, so just put this code like @godon019 's [addFriendMutation's refetchQueries style](https://github.com/geoseong/hackatalk-mobile/blob/feat%2Fusermgmt/src/components/shared/ProfileModal.tsx#L127).

please refer to [apollo's document](https://www.apollographql.com/docs/react/api/react-hooks/#options-2) how to use `refetchQueries` option.

```suggestion
  }] = useMutation<{ id: string }, MutationDeleteFriend>(MUTATION_DELETE_FRIEND, {
    refetchQueries: () => [{ query: QUERY_FRIENDS }],
  });
```